src/utils/tagsConfig.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import validUrl from './validUrl';
+
+// This is the configuration file for the tag value transformation feature.
+export default class tagsConfig extends React.Component {
```

Comment:
Should use PascalCase for class names

Reply:
Does it need to be inherited from React.Component?

Reply:
Instead of a single class, think about declaring a base class/interface, maybe with no-op implementation of transform() function. There could be a number of implementations of this interface including a default one.

A wrapper/executer class (eg. TagValuesTransformer) also has transform() which uses `tagKey -> transformer` map to do value transformations. This way, multiple tagKeys can reuse common transformers. Eg. check and cross can be reused by multiple tag keys.

It can be class hierarcy based approach or even simple objects with duck typing will work.